### ISSUE TITLE
No Items to display 

### ISSUE THREAD

**Issue by andriux1990** [opensourcepos/opensourcepos#2179]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 
- OSPOS git commit hash is: 
- PHP version is: 
- MySQL or MariaDB version is: 
- OS and version is: 
- WebServer is: 
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### Issue / Bug  / Question / New Feature

Please write your issue here. If a bug, please make sure to provide as much information as possible including configuration settings (e.g. Decimals set, Tax mode), language and steps to reproduce the bug.
__________________________________________________________________
The latest version of ospos Github does not show the search results of items in the item section .. it says "No Item to display"

**Comment by odiea** [comment]
Thanks. You are correct the current unstable build does not show items when searched.

**Comment by odiea** [comment]
This is correct for the Attributes branch I will test the Master and see what happens.

**Comment by odiea** [comment]
And this is the error.
Column 'category' in where clause is ambiguous

SELECT `items`.`item_id` AS `item_id`, MAX(items.name) AS name, MAX(items.category) AS category, MAX(items.supplier_id) AS supplier_id, MAX(items.item_number) AS item_number, MAX(items.description) AS description, MAX(items.cost_price) AS cost_price, MAX(items.unit_price) AS unit_price, MAX(items.reorder_level) AS reorder_level, MAX(items.receiving_quantity) AS receiving_quantity, MAX(items.pic_filename) AS pic_filename, MAX(items.allow_alt_description) AS allow_alt_description, MAX(items.is_serialized) AS is_serialized, MAX(items.pack_name) AS pack_name, MAX(items.deleted) AS deleted, MAX(items.custom1) AS custom1, MAX(items.custom2) AS custom2, MAX(items.custom3) AS custom3, MAX(items.custom4) AS custom4, MAX(items.custom5) AS custom5, MAX(items.custom6) AS custom6, MAX(items.custom7) AS custom7, MAX(items.custom8) AS custom8, MAX(items.custom9) AS custom9, MAX(items.custom10) AS custom10, MAX(suppliers.person_id) AS person_id, MAX(suppliers.company_name) AS company_name, MAX(suppliers.agency_name) AS agency_name, MAX(suppliers.account_number) AS account_number, MAX(suppliers.deleted) AS deleted, MAX(inventory.trans_id) AS trans_id, MAX(inventory.trans_items) AS trans_items, MAX(inventory.trans_user) AS trans_user, MAX(inventory.trans_date) AS trans_date, MAX(inventory.trans_comment) AS trans_comment, MAX(inventory.trans_location) AS trans_location, MAX(inventory.trans_inventory) AS trans_inventory, MAX(item_quantities.item_id) AS qty_item_id, MAX(item_quantities.location_id) AS location_id, MAX(item_quantities.quantity) AS quantity FROM `ospos_items` AS `items` LEFT JOIN `ospos_suppliers` AS `suppliers` ON `suppliers`.`person_id` = `items`.`supplier_id` JOIN `ospos_inventory` AS `inventory` ON `inventory`.`trans_items` = `items`.`item_id` JOIN `ospos_item_quantities` AS `item_quantities` ON `item_quantities`.`item_id` = `items`.`item_id` WHERE `location_id` = '1' AND DATE_FORMAT(trans_date, "%Y-%m-%d") BETWEEN '2010-01-01' AND '2018-10-13' AND ( `name` LIKE '%hp%' ESCAPE '!' OR `item_number` LIKE '%hp%' ESCAPE '!' OR `items`.`item_id` LIKE '%hp%' ESCAPE '!' OR `company_name` LIKE '%hp%' ESCAPE '!' OR `category` LIKE '%hp%' ESCAPE '!' ) AND `items`.`deleted` = 0 AND `items`.`item_type` IN(0, 1, 2) GROUP BY `items`.`item_id` LIMIT 25

Filename: C:/wamp64/www/newmaster/vendor/codeigniter/framework/system/database/DB_driver.php

Line Number: 691

**Comment by andriux1990** [comment]
Wow @odiea.. Thank you very much, the search works well for me now. How did you know that the problem was there, in the category?
